src/PantherTestCaseTrait.php

```diff
@@ -161,12 +161,17 @@ protected static function createPantherClient(array $options = [], array $kernel
             }
         }
 
+        // Only allow browser arguments if a browser has been explicitly specified
+        if (\is_array($browserArguments) && !\array_key_exists('browser', $options)) {
```

Comment:
If no browser is specified and browser arguments are set this should result in an exception to let the user know, these browser arguments will not be taken into account, WDYT?

Reply:
I agree, we might raise an exception. Although, I'm not sure of the right wording or exception subclass.

Reply:
I'm triggering a warning now.